Dangerfile

```diff
@@ -3,12 +3,6 @@
 rubocop.lint inline_comment: true
 
 periphery.binary_path = 'bin/periphery'
-periphery.process_warnings do |_path, _line, _column, message|
-  !message.match(/unused/i)
+periphery.scan(project: 'spec/support/fixtures/test.xcodeproj', schemes: 'test', targets: 'test') do |violation|
+  !violation.mesage.include?('unused')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/5446532a74e76ddd206b5bb0c287ca96749bfe21/Dangerfile#L7"></span>Do not leave space between <code>!</code> and its argument.</td>
    </tr>
  </tbody>
</table>